.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -15,28 +15,46 @@ This link describes the WinUI feature/API proposal process:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/feature_proposal_process.md
 -->
 
-<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 # Proposal: [your title here] 
+<!-- Add a title for your feature or API proposal. Please be short and descriptive -->
 
 ## Summary
-<!-- Please include a couple sentences summarizing your feature or API proposal -->
+<!-- Include 1-2 sentences summarizing your feature or API proposal -->
 
 ## Rationale
-<!-- Please describe WHY the feature should be added to WinUI for all developers and users.
-If applicable you can also describe how it aligns to the current WinUI roadmap and priorities:
+<!-- Fill in the table below to describe WHY the feature should be added to WinUI for all developers and users.
+Proposals often have multiple motives for why we should do the work, so list each one as a separate row item.
+If applicable you can also describe how it aligns to the current WinUI roadmap and priorities in a separate paragraph:
 https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md
 -->
+| Motive |
```

Comment:
A bulleted list might be clearer/easier than a table, especially since it's just one column

Reply:
But how will I ever satisfy my eternal thirst for tables!?!?

...ok, I yield.  I turned it into a bulleted list.  :)  Take a look and see if that's better.